containers/src/Kokkos_ScatterView.hpp

```diff
@@ -1095,7 +1095,8 @@ class ScatterView<DataType, Kokkos::LayoutLeft, ExecSpace, Op,
 
  protected:
   template <typename... Args>
-  inline original_reference_type at(int thread_id, Args... args) const {
+  KOKKOS_FORCEINLINE_FUNCTION original_reference_type at(int thread_id,
+                                                         Args... args) const {
```

Comment:
Because there were warnings showing up when it was missing, and it's clear the intent was for it to be `__host__ __device__`, since all of the other versions of `at()` in the same file are

Reply:
(Sorry, this was unrelated to MDRange cleanup, but it was such a small drive-by that I thought I should just include it)

Reply:
One-line PRs are ok and are likely to make it into `develop` faster.

Reply:
There was no issue for this, but its presence made it impossible to debug my own warnings, and I didn't think breaking it again was the right thing to do

Reply:
Fixed in 7ab72ea175.